Bottom: 4e06b971ed9218854b4f0a240c54f80e63d89ac4
Top:    7adb18dd45bd8f54eb00c14a9e5c6197722eb6eb
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-16 00:52:09 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
